rpsl-parser/examples/main.go

```diff
@@ -0,0 +1,55 @@
+package main
+
+import (
+	"fmt"
+	"os"
+
+	rpsl "github.com/manrs-tools/contrib/rpsl-parser"
+)
+
+const (
+	irrFile = "/tmp/radb.db"
+)
+
+func main() {
+	// Open the irr file, creating an io.Reader for the rpsl.Reader{}.
+	fd, err := os.Open(irrFile)
+	if err != nil {
+		fmt.Printf("Failed to open irrFile(%v): %v\n", irrFile, err)
+		return
+	}
+
+	// Make a new rpsl.Reader{}.
+	rdr := rpsl.NewReader(fd)
+
+	// Read the first character from the file.
+	// Use this character to validate that the file MIGHT be correct.
```

Comment:
is this check necessary, if you are anyway checking that the first char is a letter below?

Reply:
This gets the character to read... it effectively reads 1 char, then unreads (resets the pointer to the where the read char was). then passes the read char (r) to the 'isletter' function.

Reply:
Ok, i guess you could avoid if you had Peek() method ...  the wrapper that defines ReadRune() behavior as Read() is a little confusing (to me).

Reply:
There is a peek(), but it's not exported.
huh, I could just export it :) I'll think on that on the flight home.

Reply:
I'll kick out an exported Peek() in the next bit.